File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/9 - 20 and 21 April 2019/02. Easter Guests.py
import math

people = int(input())
budget = float(input())

count_easter_bread = math.ceil(people / 3)
count_eggs = people * 2
easter_bread = (math.ceil(people / 3)) * 4
eggs = (people * 2) * 0.45

total = easter_bread + eggs

if budget >= total:
    print(f"Lyubo bought {count_easter_bread} Easter bread and {count_eggs} eggs.")
    print(f"He has {abs(budget - total):.2f} lv. left.")
else:
    print("Lyubo doesn't have enough money.")
    print(f"He needs {abs(total - budget):.2f} lv. more.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/01. Online Exam - 20 and 21 April 2019/02. Easter Guests.js
function easterGuests(params) {
    let countPeople = Number(params.shift());
    let budget = Number(params.shift());

    let countEasterBreads = Math.ceil(countPeople / 3);
    let countEggs = countPeople * 2;

    let price = (countEasterBreads * 4) + (countEggs * 0.45);

    if (budget >= price) {
        console.log(`Lyubo bought ${countEasterBreads} Easter bread and ${countEggs} eggs.`);
        console.log(`He has ${(budget - price).toFixed(2)} lv. left.`)
    } else {
        console.log(`Lyubo doesn't have enough money.`);
        console.log(`He needs ${(price - budget).toFixed(2)} lv. more.`);
    }
}

easterGuests(['10', '35']);  // Expected Output:
// Lyubo bought 4 Easter bread and 20 eggs.
// He has 10.00 lv. left.

easterGuests(['9', '12']);  // Expected Output:
// Lyubo doesn't have enough money.
// He needs 8.10 lv. more.
